ideascube/mediacenter/templates/mediacenter/index.html

```diff
@@ -34,5 +34,5 @@
     {% include 'search/order_by.html' %}
     {% include 'search/filter_by_kind.html' %}
     {% include 'search/filter_by_lang.html' %}
-    {% tag_cloud url='mediacenter:index' model=view.model tags=available_tags %}
+    {% tag_cloud url='mediacenter:index' model=view.model limit=40 tags=available_tags %}
```

Comment:
Hum, with so much tags, yes, change the limit to 40 will not help a lot :/

I agree with you on your other remarks. I especially like the "frecent" order, but it may be tricky to implement.

We really have to do something about this, all the easy solutions we've got do not fix anything on this point and it is a real problem for users.